package.json

```
@@ -59,8 +59,10 @@
   },
   "prettier": {
     "importOrder": [
+      "<BUILTIN_MODULES>",
       "^react$",
```

oops, maybe next should be part of the built-in group
```suggestion
      "^react$",
      "^next$",
```